init.lua

```diff
@@ -73,21 +73,27 @@ armor_monoid.monoid = player_monoids.make_monoid({
 
 		join_handled[player:get_player_name()] = true
 
-		player:set_armor_groups(final)
+		if player then
```

Comment:
Useless check. `player:get_player_name()` already requires `player` to be a valid ObjectRef.